### ISSUE TITLE
Fatal error: Call to a member function getTargetEntityTypeId() 

### ISSUE THREAD

**Issue by mkstayalive** [webflo/inline_entity_form#31]
I am getting this error on using "Inline entity form - Multiple values" widget. 

_Fatal error: Call to a member function getTargetEntityTypeId() on array in /Users/manish/projects/spirituality_course/modules/inline_entity_form/src/Plugin/Field/FieldWidget/InlineEntityFormMultiple.php on line 474_

If I do a git checkout 2dbd72ce2189bcaa9a612237a438f1ed70daff4a then it works fine for me.

So, the problem seems to be in this change: ce20e4ab55795b3d69aea967c41e7e369c88c39b

**Comment by webflo** [comment]
Yes you are right. We have to change `getStorage` to `get`. Please submit a PR if it works for you.
